src/Wellcome.Dds/Wellcome.Dds.Repositories/DdsContext.cs

```diff
@@ -117,6 +121,18 @@ public IEnumerable<ValueAggregationResult> GetAggregation(string aggregator, str
             }
         }
 
+        public int GetAggregationCount(string aggregator, string value)
+        {
+            var query =
```

Comment:
Really wanted to factor this query expression out to be shared by this and the method above - but how do you do that?

Reply:
I think you'd need to switch to use LINQ methods chain rather than linq and extract a `Func`.

```cs
public IEnumerable<ValueAggregationResult> GetAggregation(string aggregator, string value, int? skip = null, int? take = null)
{
    Func<Metadata, Manifestation, bool> q = (metadata, manifestation) =>
        manifestation.Index == 0 && metadata.Label == aggregator && metadata.Identifier == value;
    
    var query = Metadata
        .Join(Manifestations, metadata => metadata.ManifestationId,
            manifestation => manifestation.PackageIdentifier,
            (metadata, manifestation) => new { metadata, manifestation })
        .Where(t1 => q(t1.metadata, t1.manifestation))
        .Select(@t => new { @t.metadata, @t.manifestation });
....
}
```

but they both use `aggregrator` and `value` so  you'd need to pass them too if it was reusable.